platform-sdk/docs/proposals/README.md

```diff
@@ -0,0 +1,150 @@
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.drawio.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be created with DrawIO and exported in SVG format. All content
+must be editable (i.e. no images). The only exception is for images that are generated by code, like the wiring diagram.
+These may be drawn on to indicate the intended changes and included as an image.
+
+If any APIs were developed in code as part of the design process, it may be included in the proposal directly or in the
+subdirectory of the proposal, but not merged into the source code of the platform.
+
+While in the draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create a new branch for the proposal.
+2. Create a subdirectory of `platform-sdk/docs/proposals/` with the name of the proposal.
+3. Copy the template `template.md` into the subdirectory and give it an appropriate name.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections or adding new sections as
+   needed. Diagrams and other files may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved while in the draft stage.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+2. Mark the PR as `Ready for Review`
+3. Update the status of the PR in
+   the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for immaterial changes can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of immaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of diagrams, etc. If any material changes are needed, the PR must be closed and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR to
+create an auditable history.
```

Comment:
Please see my response on the previous PR where you made this comment.

We are going to it this way as a starting point.

Reply:
Do we need a new branch? Or just a new PR?

Reply:
I acknowledge that you have the authority to overrule this vote (and so it's not a true veto). But I don't think I can vote anything but -1 in this circumstance.

Reply:
Our process must be optimized for efficiency in building understanding and gaining rough consensus.

On the one hand, having to take a bunch of extra steps to close a PR, open a new one, go to another page, mark one as superseded by the other and put one in draft or voting stage, is not something I'd really want to do. It seems like some of that could be optimized out.

On the other hand, the friction of taking those steps will cause me to put things in draft before it goes to vote, and have to really work on the design and building consensus **before** it goes to a vote. And that means I am more likely to put the extra effort in and actually make it easier and more productive for people to review and vote. Hopefully this is not the first time they've seen it and they're already primed to vote +1 before I even convert it from draft to ready for review.

If that is true, then maybe the extra friction of closing a failed PR isn't that big of a deal, because it shouldn't happen very often if I only moved to vote after the draft had been widely reviewed.

So I think if we put our energy into good drafts and good conversations on drafts then the voting stage should be mostly a formal affair, and not where most conversation will happen.